rust/bridge/shared/src/ffi/storage.rs

```diff
@@ -194,12 +197,12 @@ pub struct FfiPreKeyStoreStruct {
 impl PreKeyStore for &FfiPreKeyStoreStruct {
     async fn get_pre_key(
         &self,
-        prekey_id: u32,
+        prekey_id: DeviceId,
```

Comment:
This is *also* not a device ID; it's a unique identifier selecting among this device's pre-keys.

Reply:
Noted in the documentation for `PreKeyId`!